scripts/merge-and-build.sh

```diff
@@ -114,6 +139,8 @@ else
   fi
 fi # End check if we are master
 
+# TODO: Origin already vendors the web console. Should we stop updating it separately for OSE?
```

Comment:
@stevekuznetsov this means that we will need a bit more git-fu to ensure the latest commits are from tito.